hack/vendor/olm-test-script/e2e-olm.sh

```diff
@@ -70,6 +70,13 @@ if [ -n "${OPERATOR_IMAGE:-}" ] ; then
   CSV=$(echo "$CSV" | sed -e "s~image:.*~image: ${OPERATOR_IMAGE}\n~" | indent ApiVersion)
 fi
 
+ci_image_prefix="registry.svc.ci.openshift.org/origin/${VERSION}"
```

Comment:
while this should work, what the script really should do is to use `${IMAGE_FORMAT}`

you'll also need to update the elasticsearch image and the oauth-proxy image

https://github.com/openshift/origin-aggregated-logging/blob/master/hack/deploy-logging.sh#L171
and
https://github.com/openshift/origin-aggregated-logging/blob/master/hack/deploy-logging.sh#L195